@gsketefian, I think that 'run_ci' is designed to check GSI when it runs. I can see the following error: CMake Error at src/gsi/CMakeLists.txt:69 (find_package):
  By not providing "Findncio.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "ncio", but
  CMake did not find one.
  Could not find a package configuration file provided by "ncio" with any of
  the following names:
    ncioConfig.cmake
    ncio-config.cmake
  Add the installation prefix of "ncio" to CMAKE_PREFIX_PATH or set
  "ncio_DIR" to a directory containing one of the above files.  If "ncio"
  provides a separate development package or SDK, be sure it has been
  installed.
-- Configuring incomplete, errors occurred!
See also "/glade/scratch/epicufsrt/jenkins/workspace/fs-srweather-app_pipeline_PR-327/build/src/GSI/src/GSI-build/CMakeFiles/CMakeError.log".
Scanning dependencies of target emcsfc_ice_blend
src/CMakeFiles/GSI.dir/build.make:90: recipe for target 'src/GSI/src/GSI-stamp/GSI-configure' failed
make[2]: *** [src/GSI/src/GSI-stamp/GSI-configure] Error 1

Do you have any idea to check it on Gaea? I don't have access to it.

--- a/Externals.cfg
+++ b/Externals.cfg
@@ -1,11 +1,9 @@
 [regional_workflow]
 protocol = git
-#repo_url = https://github.com/ufs-community/regional_workflow
-repo_url = https://github.com/chan-hoo/regional_workflow
+repo_url = https://github.com/ufs-community/regional_workflow
 # Specify either a branch name or a hash but not both.
 #branch = develop
-#hash = 2e336c1
-branch = feature/update_hash
+hash = ac120a0
 local_path = regional_workflow
 required = True
 

--- a/docs/UsersGuide/source/ConfigWorkflow.rst
+++ b/docs/UsersGuide/source/ConfigWorkflow.rst
@@ -407,7 +407,6 @@ CCPP Parameter
    | ``"FV3_GFS_2017_gfdlmp_regional"``
    | ``"FV3_GFS_v15p2"``
    | ``"FV3_GFS_v15_thompson_mynn_lam3km"``
-   | ``"FV3_RRFS_v1alpha"``
 
 Stochastic Physics Parameters
 ================================

--- a/modulefiles/srw_common
+++ b/modulefiles/srw_common
@@ -31,4 +31,4 @@ module load wgrib2/2.0.8
 
 module load w3emc/2.9.2
 module load wrf_io/1.2.0
-#module load ncio/1.1.2
+module load ncio/1.1.2

